DeezyMatch/utils.py

```diff
@@ -84,6 +84,9 @@ def eval_map(list_of_list_of_labels,list_of_list_of_scores,randomize=True):
 
 # ------------------- string_split --------------------
 def string_split(x, tokenize=["char"], min_gram=1, max_gram=3):
+    
+    word_splitters = [" ","_"]
```

Comment:
Could you make the word splitters an argument that the function `string_split` takes as input? This way the user will be able to express more splitters if they want to

Reply:
Could you make the word splitters an argument that the function `string_split` takes as input? This way the user will be able to express more splitters if they want to